go.sum

```diff
@@ -906,8 +906,12 @@ github.com/tenntenn/text/transform v0.0.0-20200319021203-7eef512accb3 h1:f+jULpR
 github.com/tenntenn/text/transform v0.0.0-20200319021203-7eef512accb3/go.mod h1:ON8b8w4BN/kE1EOhwT0o+d62W65a6aPw1nouo9LMgyY=
 github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2 h1:mbAskLJ0oJfDRtkanvQPiooDH8HvJ2FBh+iKT/OmiQQ=
 github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2/go.mod h1:2PfKggNGDuadAa0LElHrByyrz4JPZ9fFx6Gs7nx7ZZU=
+github.com/tiancaiamao/client-go/v2 v2.0.0-alpha.0.20220804133722-ccf3e311f9e8 h1:HUmPan9oqqLIo3goZ2Gp0WSeLvGj8w7U7hVqTlcr7j0=
+github.com/tiancaiamao/client-go/v2 v2.0.0-alpha.0.20220804133722-ccf3e311f9e8/go.mod h1:v3DEt8LS9olI6D6El17pYBWq7B28hw3NnDFTxQHDLpY=
```

Comment:
Please run `go mod tidy` to clean this up.

Reply:
done